inline-demo.html

```diff
@@ -47,6 +47,9 @@
   var demoId;
 
   function receiveMessage(event) {
+    if (!event.origin.match(/^https:\/\/\w*\.?webcomponents\.org/))
```

Comment:
Can we just test that the data is in the correct format? That is has `id` and `source` properties.